src/main/java/spitter/web/services/AccountService.java

```diff
@@ -23,33 +24,33 @@
 @Component
 public class AccountService {
     private List<AccountModel> users;
-    private List<CourseRegister> courseRegisters;
+    private List<Encrollment> encrollments;
     public AccountService(){
         users = new ArrayList<>();
         AccountModel account = new AccountModel();
-        account.setStrID("qwe");
-        account.setStrPass("qwe");
-        account.setStrName("asd");
-        account.setStrEmail("qwe@qwe.com");
+        account.setID("qwe");
+        account.setPass("qwe");
+        account.setName("asd");
+        account.setEmail("qwe@qwe.com");
         users.add(account);
-        courseRegisters = new ArrayList<CourseRegister>();
-        courseRegisters.add(new CourseRegister(account,new Course(1, "JavaScript", "JavaScript is a programming language used to make web pages interactive. ")));
-        courseRegisters.add(new CourseRegister(account, new Course(2, "HTML", "HTML is a computer language devised to allow website creation.")));
-        courseRegisters.add(new CourseRegister(account, new Course(3, "JQuery", "JQuery is a programming language used to make web pages interactive")));
-        courseRegisters.add(new CourseRegister(account, new Course(4, "CSS", "CSS is a computer language devised to allow website creation.")));
-        courseRegisters.get(0).setApproved(true);
-        courseRegisters.get(3).setApproved(true);
+        encrollments = new ArrayList<Encrollment>();
+        encrollments.add(new Encrollment(1,account,new Course(1, "JavaScript", "JavaScript is a programming language used to make web pages interactive. ")));
+        encrollments.add(new Encrollment(2,account, new Course(2, "HTML", "HTML is a computer language devised to allow website creation.")));
+        encrollments.add(new Encrollment(3,account, new Course(3, "JQuery", "JQuery is a programming language used to make web pages interactive")));
+        encrollments.add(new Encrollment(4,account, new Course(4, "CSS", "CSS is a computer language devised to allow website creation.")));
+        encrollments.get(0).setStatus(EncrollStatus.APPROVED);
+        encrollments.get(3).setStatus(EncrollStatus.APPROVED);
     }
 
```

Comment:
All list of data (List<AccountModel>, List<Enrollment>, List<Course>,...) should be moved to another service, ex: `DatabaseService`. Other services (AccountService, CourseService) will depend on it.